opentelemetry/proto/common/v1/common.proto

```diff
@@ -21,6 +21,36 @@ option java_package = "io.opentelemetry.proto.common.v1";
 option java_outer_classname = "CommonProto";
 option go_package = "go.opentelemetry.io/proto/otlp/common/v1";
 
+// inlined parquet-gen options
+import "google/protobuf/descriptor.proto";
+
+message MessageOptions {
+  // As long as table_name is not blank,
+  // a schema is generated for top-level messages in each file.
+  string table_name = 1;
+}
+
+extend google.protobuf.MessageOptions {
+  MessageOptions message_opts = 1023;
+}
+
+enum TimestampType {
```

Comment:
How will `TimestampType` be used? It seems to be not used in this PR.

Reply:
it's not used right now but it's the first option inherited by the parquet generator (`1` below) and therefore I figured it would make sense to bring it along so `is_json` is not strangely assigned `2`.

Reply:
I would assume the timestampType will come handy pretty quick too.